hqdm/src/main/java/uk/gov/gchq/magmacore/hqdm/pojo/Top.java

```diff
@@ -55,10 +55,18 @@ public interface Top {
     /**
      * Get predicate value(s) by predicate Object.
      *
-     * @param predicateId Predicate ID.
+     * @param predicateId Predicate ID. {@link IRI}
```

Comment:
```suggestion
     * @param predicateIri Predicate {@link IRI}.
```

Reply:
I think these changes have gone in on the `191-thingaddvalue-is-not-tight-enough` branch as part of a separate task.

Reply:
Resolved in new commit.